src/components/Deposit/buttonState.ts

```diff
@@ -47,7 +48,13 @@ export const getState = ({
     inputAmountY,
     selectedPool?.y.asset.decimals ?? 0,
   );
-  const fee = Number(minerFee) + dexFee;
+  const minerFeeNErgs = parseUserInputToFractions(minerFee, ERG_DECIMALS);
+  const dexFeeNErgs = parseUserInputToFractions(String(dexFee), ERG_DECIMALS);
+  const feeNErgs = minerFeeNErgs + dexFeeNErgs;
+
+  const nErgsAvailable = BigInt(ergBalance || '0');
+
+  const native = selectedPool ? isNative(selectedPool.x.asset) : false;
```

Comment:
Call boolean const with `is` prefix
This expression can be simplified:
```
const isNativeAsset = isNative(selectedPool.x.asset || null);
```
ofc if it works for the `isNative` abstraction